source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.h

```diff
@@ -0,0 +1,18 @@
+//
+//  ACRPresentationViewController.h
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface ACRPresentationViewController : UIViewController
+
+@property (readonly) NSString* jsonStr;
+
+- (id) init:(NSString*) str;
```

Comment:
thanks

---
In reply to: [122546099](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122546099) [](ancestors = 122546099)